docs/source/use-monitoring/advisor/heavyCompaction.rst

```diff
@@ -1,13 +1,16 @@
 Compaction takes lots of memory and CPU
 ---------------------------------------
-Scylla runs compaction periodically as a background process. While running compaction is important, there are situations that
-the compaction takes too much CPU.
-In those cases the compaction would impact the overall system performance.
+ScyllaDB runs compaction periodically as a background process. While running compaction is important, there are situations when
+compaction takes too much CPU.
+As a result, compaction impacts the overall system performance.
 
-When facing this problem, you can statically limit the compaction shares with one of two options:
+If this is the case, you can do one of the following:
 
-1. Change ``scylla.yml`` to have ``compaction_static_shares: 100`` 
+* Statically limit the compaction shares with the ``compaction_static_shares`` option by setting a value between 50 and 1000:
 
```

Comment:
In addition to saying that the user can set optionally a value between 50 and 1000. we can say that 100 may be a good value to start with. additionally say that if read latency is impacted, it means that compaction is running too slow, then the user will have to increase compaction_static_shares until a sweet spot is reached